### ISSUE TITLE
Add XLIFF log for translation managers

### ISSUE THREAD

**Issue by timoludwig** [Integreat/integreat-cms#649]
### Motivation

At the moment, the translation managers have to manually keep track which XLIFFs they exported/imported and when.

### Proposed Solution

Track these events in the CMS and show a list of the past XLIFF actions either below the page tree inside a spoiler (which is collapsed per default) or on an extra page.

**Comment by svenseeberg** [comment]
We could actually make this part of a larger log functionality. We could, for example, show page edit events, removed pages, etc as well in this log. Do we want to have a log model or do we derive the log from our version keeping of pages? In this case we would probably need to keep track of exported XLIFFs in a new model.

**Comment by timoludwig** [comment]
> We could actually make this part of a larger log functionality. We could, for example, show page edit events, removed pages, etc as well in this log. Do we want to have a log model or do we derive the log from our version keeping of pages? In this case we would probably need to keep track of exported XLIFFs in a new model.

Good idea! Moving and deleting pages are events which cannot be derived from our current models, so adding an additional log model with different categories could be reasonable.

**Comment by timoludwig** [comment]
Interesting: There is already a [LogEntry](https://github.com/django/django/blob/main/django/contrib/admin/models.py) model in `django.contrib.admin`...
Using this log has the advantage that all actions are also visible in the django admin UI for debugging... I'm just not sure if we can use it when we deactivate the django admin in production mode.

**Comment by laravoelk** [comment]
I will ask and get back to you soon. :-)

**Comment by laravoelk** [comment]
I have spoken to Vijay about this: 
If it is not a big effort on the part of the CMS team to continue to run the logs, it would make sense....
as long as the municipalities are still working with XLIFF exports
